docs/source/using/experimental-features.adoc

```diff
@@ -33,10 +33,8 @@ You can set the `MINISHIFT_ENABLE_EXPERIMENTAL` environment variable to enable t
 `extra-clusterup-flags`::
 Enables passing flags directly to `oc cluster up` that are not directly exposed in the {project} CLI.
 
-For example, the following command will pass the `--service-catalog` flag to `oc cluster up` to provision the OpenShift link:https://docs.okd.io/latest/architecture/service_catalog/index.html[*service catalog*]:
-
 ----
-$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--service-catalog"
+$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog'"
```

Comment:
This method works for okd v3.10 the earlier one works for v3.9 we need to mention that and keep both the ways in the docs.

Reply:
Good point! I will update it